.github/workflows/build.yaml

```diff
@@ -35,24 +35,30 @@ jobs:
           format: 'table'
           ignore-unfixed: true
           vuln-type: 'os,library'
+      - name: Trivy - List all vulnerabilities
```

Comment:
You have this action twice now, you should remove the other one or the build fails